truss/server/control/truss_patch/model_code_patch_applier.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+from pathlib import Path
+
+from truss.server.control.patch.types import Action, ModelCodePatch
+
+
+def apply_code_patch(
+    relative_dir: Path,
+    patch: ModelCodePatch,
+    logger: logging.Logger,
+):
+    logger.debug(f"Applying code patch {patch.to_dict()}")
+    filepath: Path = relative_dir / patch.path
+    action = patch.action
```

Comment:
Nit: This local var seems unneeded.

Reply:
This is part of the code I'm porting over. Thinking to keep the clean up as a separate PR to make things less complciated.